scripts/monitor/notebook/plot-cromwell-bq-monitoring.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Explain to the users here that two scenarios are shown:</p><ul><li>a fresh query, which we then turn around and save it locally</li><li>a query that has been run and saved before, where we simply reload the data</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/broadinstitute/long-read-pipelines/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>done</p>